app/assets/stylesheets/views/_transition-landing-page.scss

```diff
@@ -168,11 +168,8 @@ $green: #00703c;
   padding: 0 3px;
 }
 
-.landing-page__page_title {
-  @include govuk-font($size: 48, $weight: bold);
-  color: $dark-blue;
+.landing-page__title-container {
```

Comment:
Potentially this could be removed and added directly on the markup using the Design system:
 `govuk-!-margin-top-7`